### PR TITLE
Releases Notes for 1.7.1

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Closes https://github.com/xbmc/Official-Kodi-Remote-iOS/issues/300.

## Summary for release notes

[not app] Release Notes 1.7.1

### CODE REVIEW

**Comment by kambala-decapitator** [comment]
what do you think about adding some note regarding TF testing to release notes? That more users are aware of it.

**Comment by wutschel** [comment]
> what do you think about adding some note regarding TF testing to release notes? That more users are aware of it.

Directly adding https://testflight.apple.com/join/VQkpfqDN? Or is the AppStore allowing to place a html link (<a href="https://testflight.apple.com/join/VQkpfqDN">Access to Beta-Software in TestFlight</a>.)

**Comment by kambala-decapitator** [comment]
probably not the link itself as it's not clickable afaik, but rather some text mentioning TF and forums.

the only URL fields there are don't look suitable for TF link:
Screenshot 2021-05-20 at 09 57 39

**Comment by kambala-decapitator** [comment]
> Ok, no hyperlink/html then. What about putting a footer like this:
> 
> > Beta-Testers are welcome to join us on TestFlight. Please follow us on https://forum.kodi.tv/forumdisplay.php?fid=193.

Yes, good idea! Please add to the PR.

> Btw, wouldn´t marketing URL be https://kodi.tv?

doesn't quite fit here I think:

> A URL with marketing information about your app. This URL will be visible on the App Store.

**Comment by wutschel** [comment]
> Yes, good idea! Please add to the PR.

Done

> > Btw, wouldn´t marketing URL be https://kodi.tv?
> 
> doesn't quite fit here I think:
> 
> > A URL with marketing information about your app. This URL will be visible on the App Store.

I think this fits quite well. Following links from this page the remote App is listen. It somehow is the Kodi home, the App is a kind of spin-off from there.

File: fastlane/metadata/en-US/release_notes.txt [hunk 1]
```diff
@@ -1,11 +1,7 @@
-Features:
-- Ignore articles based on Kodi sort tokens (e.g. "The")
-- Introduce mute functionality
-- Improved iPad fullscreen mode for Music
-
 Bugfixes:
-- Fix Wake-On-LAN on iOS 14.5
+- Fix sorting issues with Add-Ons (e.g. youtube)
+- Fix volume up/down button function for iPhone
```
Comment (kambala-decapitator) [inline fastlane/metadata/en-US/release_notes.txt:3]: please make it the first item

File: fastlane/metadata/en-US/release_notes.txt [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
+- Fix long-press action in iPad fullscreen mode for Music
 - Further bugfixes and stability improvements
 
-Other:
-- Maintenance, refactoring and code quality improvements
+Beta-Testers are welcome to join us on TestFlight. Please follow us on https://forum.kodi.tv/forumdisplay.php?fid=193.
```
Comment (kambala-decapitator) [inline fastlane/metadata/en-US/release_notes.txt:7]:
```suggestion
Beta-testers are welcome to join TestFlight! Please follow us on https://forum.kodi.tv/forumdisplay.php?fid=193
```

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

**Comment by kambala-decapitator** [comment]
> I think this fits quite well. Following links from this page the remote App is listen. It somehow is the Kodi home, the App is a kind of spin-off from there.

ok :) then please add the URL to `fastlane/metadata/en-US/marketing_url.txt`

File: fastlane/metadata/en-US/release_notes.txt (hunk 2 above)
Comment (wutschel) [reply fastlane/metadata/en-US/release_notes.txt]: Done

File: fastlane/metadata/en-US/release_notes.txt (hunk 1 above)
Comment (wutschel) [reply fastlane/metadata/en-US/release_notes.txt:3]: Sorry, I don't understand your comment.

Comment (wutschel) [reply fastlane/metadata/en-US/release_notes.txt:2]: :) Done

**Pull request merged**